library/src/main/java/com/chuckerteam/chucker/internal/support/OkHttpUtils.kt

```diff
@@ -51,18 +54,23 @@ private val Headers.containsGzip: Boolean
         return this["Content-Encoding"].equals("gzip", ignoreCase = true)
     }
 
-private val supportedEncodings = listOf("identity", "gzip")
+private val Headers.containsBrotli: Boolean
+    get() {
+        return this["Content-Encoding"].equals("br", ignoreCase = true)
+    }
+
+private val supportedEncodings = listOf("identity", "gzip", "br")
 
 internal val Headers.hasSupportedContentEncoding: Boolean
     get() = get("Content-Encoding")
         ?.takeIf { it.isNotEmpty() }
         ?.let { it.toLowerCase(Locale.ROOT) in supportedEncodings }
         ?: true
 
-internal fun Source.uncompress(headers: Headers) = if (headers.containsGzip) {
-    gzip()
-} else {
-    this
+internal fun Source.uncompress(headers: Headers) = when {
+    headers.containsGzip -> gzip()
+    headers.containsBrotli -> BrotliInputStream(this.buffer().inputStream()).source()
```

Comment:
I wonder if we should add a list of supported compression types in `ChuckerInterceptor` KDoc. I'm saying that because `BinaryDecoder` mentions that it the body will be gunzipped but now we also understand Brotli. This way in `BinaryDecoder` KDoc we could just mention that body will be uncompressed and to see which compression types are supported see `ChuckerInterceptor`.

Reply:
Good catch. We should mention in KDoc or add something in our Readme, because now it is not clear for end users that Chucker supports compressed payloads, so even having Gzip is something unclear.